File: Open Java SDK/src/main/java/com/boticz/open/client/OpenClient.java
package com.boticz.open.client;

import com.boticz.open.service.IotDeviceService;

public class OpenClient {
    private final IotDeviceService iotDevice;

    public OpenClient(Config config) {
        OpenHttpClient httpClient = new OpenHttpClient(config);
        this.iotDevice = new IotDeviceService(httpClient);
    }

    public IotDeviceService getIotDevice() {
        return iotDevice;
    }
}

File: open-python-sdk/boticz_open/open_client.py
from .client import Client
from .config import Config
from .iot_device import IotDeviceService


class OpenClient:
    def __init__(self, base_url: str, token: str, session=None):
        client = Client(Config(base_url=base_url, token=token, session=session))
        self.IotDevice = IotDeviceService(client)

